app/models/create_expenses_with_receipt.rb

```diff
@@ -0,0 +1,29 @@
+class CreateExpensesWithReceipt
+  def initialize(expense_params)
+    @params = expense_params
+  end
+
+  def create_expense_and_receipt
```

Comment:
I'm not a big fan of this name, because it is a bit redundant when compared to the class name. Instead, I would just call it `create`.

Or, if you want to make look more like a pattern thing, and make it clearer that it will only have one public method, it can be called `call`, `run`or `process`.

What do you think?